### PR TITLE
Fix CI build

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

#### What it does

Fix CI build.

#### Review checklist

- [ ] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by olexii4** [comment]
I can't reproduce it locally. So, I am going to test it with CI.

**Comment by marcdumais-work** [comment]
Hi @olexii4 . FYI the repo was just moved to the `eclipse-theia` organization and CI is not completely configured yet. so issues are expected.

**Comment by kittaakos** [comment]
> I can't reproduce it locally

I could not reproduce it locally either.

> FYI the repo was just moved to the `eclipse-theia` organization and CI is not completely configured yet

Perfect timing 😄. Shall we revert the PR?

**Comment by kittaakos** [comment]
> I can't reproduce it locally.

Nice that you found the fix, but can you explain why it worked locally and not from the CIs?

**Comment by olexii4** [comment]
@kittaakos I cannot tell exactly(ideas only). I just fixed an error from the CI log.

**Comment by kittaakos** [comment]
> > I can't reproduce it locally
> 
> I could not reproduce it locally either.

It was a test failure. We do not run the tests :/

> I cannot tell exactly(ideas only). 

I did not know it either. 

When we load the `connection-status-service.spec` module with `mocha` the following modules will be required and loaded: `connection-status-service` > `frontend-application` > `shell/shell-layout-restorer` > `theming`. The required object exists. It cannot be an incorrect CSS file path or config issue in `mocha`.

```
require('../../src/browser/style/materialcolors.css')
Object {}
[[StableObjectId]]:8
__proto__:Object {constructor: , __defineGetter__: , __defineSetter__: , …}
__defineGetter__:function __defineGetter__() { … }
__defineSetter__:function __defineSetter__() { … }
__lookupGetter__:function __lookupGetter__() { … }
__lookupSetter__:function __lookupSetter__() { … }
[[StableObjectId]]:9
constructor:function Object() { … }
hasOwnProperty:function hasOwnProperty() { … }
isPrototypeOf:function isPrototypeOf() { … }
propertyIsEnumerable:function propertyIsEnumerable() { … }
toLocaleString:function toLocaleString() { … }
toString:function toString() { … }
valueOf:function valueOf() { … }
__proto__:null
```
(_from debug console ☝️_)

However, the [`style-loader`](https://github.com/webpack-contrib/style-loader) does not work in `mocha` tests.

File: packages/core/src/browser/theming.ts [hunk 1]
```diff
@@ -23,8 +23,6 @@ import { FrontendApplicationConfigProvider } from './frontend-application-config
 import { MenuContribution, MenuModelRegistry } from '../common/';
 import { CommonMenus } from './common-frontend-contribution';
 
-require('../../src/browser/style/materialcolors.css').use();
```
Comment (akosyakov) [inline packages/core/src/browser/theming.ts]: please move it to the same place where index.css is imported in core

**Pull request merged**